stdlib/public/core/NativeDictionary.swift

```diff
@@ -468,6 +468,20 @@ extension _NativeDictionary { // Insertions
   }
 }
 
+extension _NativeDictionary {
+  @inlinable
+  internal mutating func swapAt(_ i: Index, _ j: Index, isUnique: Bool) {
+    let rehashed = ensureUnique(isUnique: isUnique, capacity: capacity)
+    _sanityCheck(!rehashed)
+    validate(i)
+    validate(j)
+    let key = (_keys + i.bucket.offset).move()
+    let value = (_values + i.bucket.offset).move()
+    moveEntry(from: j.bucket, to: i.bucket)
+    uncheckedInitialize(at: j.bucket, toKey: key, value: value)
```

Comment:
Uhh, on second though, maybe I shouldn't swap the keys?

I'll just quietly force push a fix and pretend this never happened... 🤦🏻‍♀️